pkg/providers/aws/provider_blobstorage.go

```diff
@@ -156,10 +157,68 @@ func (p *BlobStorageProvider) CreateStorage(ctx context.Context, bs *v1alpha1.Bl
 		},
 	}
 
+	// Adding tags to s3
+	_, err = p.TagBlobStorage(ctx, *bucketCreateCfg.Bucket, bs, stratCfg.Region)
+	if err != nil {
+		return nil, msg, errorUtil.Wrapf(err, string(msg))
+	}
+
 	p.Logger.Infof("creation handler for blob storage instance %s in namespace %s finished successfully", bs.Name, bs.Namespace)
 	return bsi, msg, nil
 }
 
+func (p *BlobStorageProvider) TagBlobStorage(ctx context.Context, bucketName string, bs *v1alpha1.BlobStorage, stratCfgRegion string) (croType.StatusMessage, error) {
+	p.Logger.Infof("bucket %s found, Adding tags to bucket", bucketName)
+
+	// get the environment from the CR ,
+	defaultOrganizationTag, exists := os.LookupEnv("DEFAULT_ORGANIZATION_TAG")
+	if exists {
+		// Print the value of the environment variable
+		p.Logger.Infof("env var DEFAULT_ORGANIZATION_TAG : %s", defaultOrganizationTag)
+	}
+
+	// new session but don't need credentials
+	s3svctag := s3.New(session.Must(session.NewSession(&aws.Config{Region: aws.String(stratCfgRegion)})))
+
+	//get Cluster Id
+	clusterId, _ := resources.GetClusterId(ctx, p.Client)
+	// Tagging input
+	for _, productName := range bs.ObjectMeta.Labels {
```

Comment:
from my reading of this, we go through each label in the kube resource, and use that label as the `<org-tag>/product-name` tag in AWS, is that correct?

if so, with a kube resource like:

```
metadata:
    name: test
    labels:
        productName: myProduct
        somethingElse: notMyProduct
```

would this perform two tag updates and result in the AWS tag being `<org-tag>/product-name=notMyProduct`?

i know the ticket discussed the idea of allowing  tags to be added arbitrarily based on the label, but I think for now we can just set `<org-tag>/product-name` to be `bs.ObjectMeta.Labels["productName"]` if it exists and isn't an empty string. then we can avoid any looping for now.

if it's an empty string, let's just not add the tag for now. if that makes sense?

Reply:
@aidenkeating something like 
```
if bs.ObjectMeta.Labels["productName"] != "" && len(bs.ObjectMeta.Labels["productName"]) > 0 {do stuff}
``` 
or would the length greater that zero be sufficient

Reply:
@austincunningham It slipped my mind that strings default to an empty string in go, so we can just check for the empty string here, like

```
// define the default list of tags
if bs.ObjectMeta.Labels["productName"] != "" {
    // add the product name tag to the list of tags
}
```

Reply:
:+1: